packages/wonder-blocks-i18n/src/index.js

```diff
@@ -9,3 +9,5 @@ export {
 } from "./functions/i18n.js";
 
 export {localeToFixed, getDecimalSeparator} from "./functions/l10n.js";
+
+export {getLocale, setLocale} from "./functions/locale.js";
```

Comment:
Whoops, think you forgot to include locale.js!

Reply:
It's there now.